### ISSUE TITLE
Design Feedback - Design Intent - VANotify Unsubscribe Page

### ISSUE THREAD

**Issue by seanasewell** [department-of-veterans-affairs/va.gov-team#32880]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
-

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
 - Manage preference should be sentence case instead of title case - update “Manage Preferences” to “Manage preferences”.

**Should:**
- Remove grey background behind content.

**Consider:**
- Treating “Manage preferences” as a text link instead of a button.
- Adding the word “email” in between “Manage” and “preferences”.
- Adding an unsuccessful unsubscribe state.

## Platform directions
- Update "Issue Title"
- Link to collab cycle touchpoint meeting epic
- Add to the Feedback pipeline on the Platform Product Support board
- Add your feedback
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by natashaj7** [comment]
In response to the second piece of feedback under the Must section
->"Treating “Manage preferences” as a text link instead of a button. If it is navigational, it cannot be a button due to accessibility issues."

My understanding after discussing this item was that it should remain a button as it directs to an authentication modal that then navigates the user to preferences.

**Comment by natashaj7** [comment]
We decided to change the language in the "Manage preferences" button to "Manage settings" in order to match the language used on VA.gov. We will run this by the content team in the content and IA intake meeting.

**Comment by natashaj7** [comment]
Can you share the link to the  Platform Product Support board so we can add to the feedback pipeline as directed above?

**Comment by natashaj7** [comment]
@seanasewell we incorporated the "Must" feedback items except for the second one which we provided feedback on, and updated this ticket

**Comment by seanasewell** [comment]
> In response to the second piece of feedback under the Must section ->"Treating “Manage preferences” as a text link instead of a button. If it is navigational, it cannot be a button due to accessibility issues."
> 
> My understanding after discussing this item was that it should remain a button as it directs to an authentication modal that then navigates the user to preferences.

Let me connect with Ryan this morning and get back to you. Thanks @natashaj7

**Comment by seanasewell** [comment]
@natashaj7 Sorry for the delayed response on this. Just to confirm, when a user clicks the unsubscribe link in an email and is met with the unsubscribe confirmation page, are they always going to be logged out? In that scenario, a "Manage settings" button makes sense since it has to take them to an authentication modal in order for them to log in first before navigating to the settings page. 

However, is there a scenario where they will already be logged in while on the unsubscribe confirmation page? In that case, "Manage settings" link makes more sense since it will navigate them directly to the settings page, bypassing the authentication modal experience.

**Comment by allison0034** [comment]
@natashaj7 sorry for the late feedback, Sean, Ryan and myself just discussed this and wanted to add...

Consider making the unsubscribe and preferences distinct, separate actions. Rely on headers to define the space.

**Comment by natashaj7** [comment]
@seanasewell I can design a page that displays that button as a link as well as having the version with a button but then the VA engineering team would need to display a different UI based on whether the user is logged in. I wonder if they are able to do so or if it's over engineering the page a little.
@allison0034 I'm struggling to visually see how your suggestion would look on this page. Can we hop on a call to chat about it?

**Comment by allison0034** [comment]
@natashaj7 yes! So something similar to this comp, however you do not need the card style, you could separate it with headers. ![image (1).png](https://images.zenhubusercontent.com/5f7e15543782a5c1ee10f1f0/bac47a57-e790-487b-8046-0ed08081c826)

**Comment by natashaj7** [comment]
Thanks Allison, we are working on incorporating and testing some of the feedback from Mikki in the IA and content review as well.

**Comment by rtwell** [comment]
I wanted to drop a couple clarifying notes about this design. Apologies for any confusion! What Allison shared is a good example of delineating actions, but it's got some semantic issues that will not meet VA.gov standards.

1. Every page on VA.gov must have 1 and only 1 `h1`. That will change the structure/order of this design a bit. 
If the `h1` of this page is "Unsubscribe successful", then "Manage notification preference" might be the `h2` (I would work with the content team if you have any questions). 
it might be something like:
`h1` Unsubscribe successful
`p` email-address@whatever.com will not…
`button` Subscribe again
`h2` Manage notification settings
`p` Sign in to your…
`button` Manage VA.gov settings
etc etc
2. Given that we now have a different structure for this page, there is no need for the gray background. These are not separate-but-equal interactions, they are primary and secondary. This should be a simple page with text, that spans the appropriate number of columns using our [flex box utility](https://design.va.gov/utilities/flexbox). 
3. See our [documentation](https://design.va.gov/components/buttons#usage) regarding the use of buttons vs links. If the actions on this page are prompting a sign in modal (even when routing to a new page), or submitting data, a button is the appropriate UI element. If we are taking users to a new pages without sign in, [links](https://design.va.gov/content-style-guide/links) or [action links](https://design.va.gov/experimental-design/action_links) would be appropriate. 
4. Because we have "sign in" above the buttons, a11y recommends using `aria-describedby` to link the additional hint text to those buttons so it's extra clear what will happen (as some screen reader users may skip those completely using the tab key).
5. I wanted to drop a couple clarifying notes about this design. Apologies for any confusion! What Allison shared is a good example of delineating actions, but it's got some semantic issues that will not meet VA.gov standards.

1. Every page on VA.gov must have 1 and only 1 `h1`. That will change the structure/order of this design a bit. 
If the `h1` of this page is "Unsubscribe successful", then "Manage notification preference" might be the `h2` (I would work with the content team if you have any questions). 
it might be something like:
`h1` Unsubscribe successful
`p` email-address@whatever.com will not…
`button` Subscribe again
`h2` Manage notification settings
`p` Sign in to your…
`button` Manage VA.gov settings
etc etc
2. Given that we now have a different structure for this page, there is no need for the gray background. These are not separate-but-equal interactions, they are primary and secondary. This should be a simple page with text, that spans the appropriate number of columns using our [flex box utility](https://design.va.gov/utilities/flexbox). 
3. See our [documentation](https://design.va.gov/components/buttons#usage) regarding the use of buttons vs links. If the actions on this page are prompting a sign in modal (even when routing to a new page), or submitting data, a button is the appropriate UI element. If we are taking users to a new pages without sign in, [links](https://design.va.gov/content-style-guide/links) or [action links](https://design.va.gov/experimental-design/action_links) would be appropriate. 
4. Because we have "sign in" above the buttons, a11y recommends using `aria-describedby` to link the additional hint text to those buttons so it's extra clear what will happen (as some screen reader users may skip those completely using the tab key).
I wanted to drop a couple clarifying notes about this design. Apologies for any confusion! What Allison shared is a good example of delineating actions, but it's got some semantic issues that will not meet VA.gov standards.

1. Every page on VA.gov must have 1 and only 1 `h1`. That will change the structure/order of this design a bit. 
If the `h1` of this page is "Unsubscribe successful", then "Manage notification preference" might be the `h2` (I would work with the content team if you have any questions). 
it might be something like:
`h1` Unsubscribe successful
`p` email-address@whatever.com will not…
`button` Subscribe again
`h2` Manage notification settings
`p` Sign in to your…
`button` Manage VA.gov settings
etc etc
2. Given that we now have a different structure for this page, there is no need for the gray background. These are not separate-but-equal interactions, they are primary and secondary. This should be a simple page with text, that spans the appropriate number of columns using our [flex box utility](https://design.va.gov/utilities/flexbox). 
3. See our [documentation](https://design.va.gov/components/buttons#usage) regarding the use of buttons vs links. If the actions on this page are prompting a sign in modal (even when routing to a new page), or submitting data, a button is the appropriate UI element. If we are taking users to a new pages without sign in, [links](https://design.va.gov/content-style-guide/links) or [action links](https://design.va.gov/experimental-design/action_links) would be appropriate. 
4. Because we have "sign in" above the buttons, a11y recommends using `aria-describedby` to link the additional hint text to those buttons so it's extra clear what will happen (as some screen reader users may skip those completely using the tab key).
5. Users can only manage _text preferences_ in profile. To manage _email preferences_, users have to go to My HealtheVet. We should make this UX clear in this design.

**Comment by natashaj7** [comment]
Hi @rtwell,
Oddball was awarded the contract to move forward with design and development on VANotify which means they will take over this process moving forward. CCing @hubgitian @miabecker 
Thanks for the collaboration and thoughtful discussion thus far. It's been really interesting to learn more about design practices at the VA through this process. Wishing all the best in this transition and to the new team stewarding this tool.
